E   TypeError: save() missing 1 required positional argument: 'sample_rate'

Hello,

Getting error in using 

`enhancer = SpectralMaskEnhancement.from_hparams(
    source="speechbrain/mtl-mimic-voicebank",
    savedir='stream_folder/',
    run_opts={"device":"cuda"})

enhanced = enhancer.enhance_file("stream_folder/fr_yourname.wav", output="stream_folder/test.wav)`

test_speechbrain.py:None (test_speechbrain.py)
test_speechbrain.py:13: in <module>
    enhanced = enhancer.enhance_file("stream_folder/fr_yourname.wav",
/home/tim/env/python3.8/lib/python3.8/site-packages/speechbrain/pretrained/interfaces.py:1929: in enhance_file
    torchaudio.save(output_filename, enhanced, channels_first=False)
E   TypeError: save() missing 1 required positional argument: 'sample_rate'

probably an update on torchaudio.save you need to handle.

Regards

--- a/speechbrain/inference/enhancement.py
+++ b/speechbrain/inference/enhancement.py
@@ -105,7 +105,11 @@ class SpectralMaskEnhancement(Pretrained):
             enhanced = self.enhance_batch(batch)
 
         if output_filename is not None:
-            torchaudio.save(output_filename, enhanced, channels_first=False)
+            torchaudio.save(
+                uri=output_filename,
+                src=enhanced,
+                sample_rate=self.hparams.compute_stft.sample_rate,
+            )
 
         return enhanced.squeeze(0)
 
@@ -169,7 +173,11 @@ class WaveformEnhancement(Pretrained):
         enhanced = self.enhance_batch(batch)
 
         if output_filename is not None:
-            torchaudio.save(output_filename, enhanced, channels_first=False)
+            torchaudio.save(
+                uri=output_filename,
+                src=enhanced,
+                sample_rate=self.audio_normalizer.sample_rate,
+            )
 
         return enhanced.squeeze(0)
 

